client/blocks/image-editor/index.jsx

```diff
@@ -71,22 +71,18 @@ class ImageEditor extends Component {
 
 	editCanvasRef = createRef();
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillReceiveProps( newProps ) {
+	componentDidUpdate( prevProps ) {
 		const { media: currentMedia } = this.props;
 
-		if ( newProps.media && ! isEqual( newProps.media, currentMedia ) ) {
+		if ( this.props.media && ! isEqual( prevProps.media, currentMedia ) ) {
```

Comment:
We could use `currentMedia` here.

Reply:
fixed via https://github.com/Automattic/wp-calypso/pull/62944/commits/27ceec7d65c1cd75020ab08c84353474cdda50a8